docs/guide/vite.md

```diff
@@ -0,0 +1,383 @@
+
+### Method 1: Module Configuration
+
+Use the `build.vite` property in your module to extend Vite's configuration:
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  build: {
+    vite: {
+      myViteConfig: {
+        // Vite configuration options
+        define: {
+          __API_URL__: JSON.stringify('https://api.example.com'),
+          __ANALYTICS_ENDPOINT__: JSON.stringify(process.env.ANALYTICS_URL)
+        },
+        css: {
+          // CSS configuration options
+          preprocessorOptions: {
+            scss: {
+              additionalData: `$injectedColor: orange;`
+            }
+          }
+        },
+        optimizeDeps: {
+          // Dependencies optimization options
+          include: ['lodash', 'moment']
+        },
+        server: {
+          // Development server options
+          https: true,
+          port: 3000
+        }
+      }
+    }
+  }
+};
+```
+  <template v-slot:caption>
+    modules/analytics/index.js
+  </template>
+</AposCodeBlock>
+
+### Method 2: Project Configuration File
+
+For project-wide Vite settings, create either `apos.vite.config.js` (ESM) or `apos.vite.config.mjs` (CJS) in your project root:
+
+<AposCodeBlock>
+
+```javascript
+import { defineConfig } from '@apostrophecms/vite/vite';
+
+export default defineConfig({
+  // All standard Vite configuration options are supported
+  plugins: [
+    // Add Vite plugins
+    svgLoader({
+      svgoConfig: {/* custom SVG optimization options */}
+    })
+  ],
+  resolve: {
+    alias: {
+      '@components': '/src/components',
+      '@styles': '/src/styles'
+    }
+  },
+  build: {
+    rollupOptions: {
+      // Customize underlying Rollup bundle
+      output: {
+        manualChunks: {
+          vendor: ['react', 'react-dom']
+        }
+      }
+    }
+  }
+});
+```
+  <template v-slot:caption>
+    apos.vite.config.js
+  </template>
+
+</AposCodeBlock>
+
+## Common Configuration Options
+
+Here are some frequently used Vite configuration options:
+
+- **define**: Replace constants in your code during build time
+- **plugins**: Add Vite plugins for additional functionality
+- **resolve.alias**: Create import aliases for cleaner imports
+- **css**: Configure CSS processing and preprocessors
+- **build**: Customize build output and bundling behavior
+- **server**: Configure development server settings
+- **optimizeDeps**: Control dependency pre-bundling
+
+For a complete list of available options, refer to the [Vite Configuration Documentation](https://vitejs.dev/config/).
+
+::: warning
+When creating aliases that reference ApostropheCMS core or module code, the proper syntax for referencing the compiled apos-build directory is still being developed. For now:
+
+* Use the built-in alias `Modules/module-name/components/...` for accessing ApostropheCMS module components
+* Project-specific aliases (pointing to your own source code) work as normal using standard Vite alias syntax
```

Comment:
Nope, they don't know the path. It should point to our build folder an not their module original code.

Reply:
For the first draft release I'm going to leave this warning out - I don't understand well enough.